programs/bpf/tests/programs.rs

```diff
@@ -812,7 +812,8 @@ fn test_program_bpf_invoke_sanity() {
     const TEST_WRITABLE_DEESCALATION_WRITABLE: u8 = 14;
     const TEST_NESTED_INVOKE_TOO_DEEP: u8 = 15;
     const TEST_EXECUTABLE_LAMPORTS: u8 = 16;
-    const TEST_RETURN_DATA_TOO_LARGE: u8 = 18;
+    const TEST_CALL_PRECOMPILE: u8 = 17;
+    const TEST_RETURN_DATA_TOO_LARGE: u8 = 19;
```

Comment:
is there any reason, the number bumped from 17 => 19? why can't `TEST_RETURN_DATA_TOO_LARGE` be `18`?

Reply:
oh, maybe because of this? https://github.com/solana-labs/solana/pull/19930/files#r717858614

Reply:
no reason, just not ordered, I'll order it to make it loo pretty :-)

Reply:
Hmm, @seanyoung , looks like maybe the data too large test case was never added to the test program so the test below passes due to an unrelated failure?
https://github.com/solana-labs/solana/commit/098585234d0051c080f2639088b924be340573d3

Reply:
I don't think it was even called. Fixed in https://github.com/solana-labs/solana/pull/20340